SPEC.md

```diff
@@ -277,6 +289,12 @@ are no new events. For instance, the publisher could be filtering away
 a lot of internal events to produce the external feed, and being able to
 make progress also in the case of no new (external) events is important.
 
+The returned cursor can in some cases be an empty string. This means consumption
```

Comment:
Here, it seems the representation of "done" is an empty string, while earlier it was mentioned it was `_done`. Which is correct?